ci/build_alpine_container/README.md

```diff
@@ -0,0 +1,17 @@
+This directory contains a Dockerfile that builds an envoy image
```

Comment:
@vijayendrabvs in that case please change this PR to make the additional push on `docker_push.sh`. I'll review when you update.